crates/sui-indexer/src/processors/package_processor.rs

```diff
@@ -56,6 +62,9 @@ impl PackageProcessor {
 
             last_processed_id += event_count as i64;
             commit_package_log(&mut pg_pool_conn, last_processed_id)?;
+            self.package_processor_metrics
```

Comment:
Why no "received" for this one?

Reply:
Similar question for the other `batch` metrics.

Reply:
> Why no "received" for this one?

processors like `package_processor` does not have fetch data from FN, instead they count on the shared DB instance, so no need to count the received one.